torch/_refs/nn/functional/__init__.py

```diff
@@ -176,35 +175,6 @@ def celu(
     return torch.where(a > 0, a, rhs)
 
 
-@register_decomposition(aten.dropout)
-@_inplace_wrapper
-@out_wrapper()
-def dropout(
```

Comment:
If this is just for the sake of export, then wouldn't it make more sense to remove `aten.dropout` from the core decomposition list but still use it in inductor?

Reply:
Oh it's composite. Thats rather annoying.

Still shouldn't delete the reference implementation though, and it's bad for inductor that we're loosing the shortcuts for `not training` and `p in (0, 1)`.

Reply:
ok, I am keeping the ref, but not registering it as decomp.